Tests/xcprojTests/XcodeProjIntegrationSpec.swift

```diff
@@ -13,28 +13,36 @@ final class XcodeProjIntegrationSpec: XCTestCase {
     }
 
     func test_init_hasTheASharedData() {
-        let got = project()
+        let got = projectiOS()
         XCTAssertNotNil(got?.sharedData)
     }
 
     func test_write() {
-        testWrite(from: fixturePath(),
+        testWrite(from: fixtureiOSProjectPath(),
                   initModel: { try? XcodeProj(path: $0) },
                   modify: { $0 })
     }
-
-    private func fixturePath() -> Path {
-        return fixturesPath() + "iOS/Project.xcodeproj"
+    
+    func test_init_usesAnEmptyWorkspace_whenItsMissing() {
+        let got = projectWithoutWorkspace()
```

Comment:
Why did you use this name?

Reply:
It's a project that doesn't have a workspace internally. Would you use a different name @artemnovichkov?